### ISSUE TITLE
end_callback function triggered on mounted.

### ISSUE THREAD

**Issue by rcordobar19** [TriDiamond/vuejs-countdown-timer#9]
So when you load the component, because of what TriDiamond stands on his pull request. (https://github.com/TriDiamond/vuejs-countdown-timer/pull/7) the end_callback functionn will be triggered.

I fixed by adding another check at the end_callback function.
**if(this.currentTime < this.endsAt) return;**

```
methods: {
            yourCallBackFunction() {
                if(this.currentTime < this.endsAt) return;
               ....
            }
        },
```

**Comment by bennyxguo** [comment]
I can fix this from the core part of the callback function trigger by adding in timing conditions. I will fix this when I get the time this week.

**Comment by bennyxguo** [comment]
@rcordobar19
Fixed this problem on v2.1.2

## 2.1.2 (Oct 07, 2019)
### Bug fixes 🐛
- Fixed reenter the active window current time is not recalculated.
- Fixed event ended-callback triggered on vue mounted.
